# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/moz.build
# Commit: 0bf045553a40
# Full Hash: 0bf045553a4079b9948e1814e5cd1038be301b0b
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2019-11-02 09:54:21
# Regressor Bug: 1581637
# File Overlap Count: 1
# Description:
#   Bug 1581637 - Part 8 - Add Http3Session/Http3Stream. r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D46652
# ==============================================================================

diff -r 2801613e5d2d -r 0bf045553a40 netwerk/protocol/http/moz.build
--- a/netwerk/protocol/http/moz.build	Fri Nov 01 14:55:56 2019 +0000
+++ b/netwerk/protocol/http/moz.build	Fri Nov 01 14:55:55 2019 +0000
@@ -77,6 +77,8 @@
     'Http2Push.cpp',
     'Http2Session.cpp',
     'Http2Stream.cpp',
+    'Http3Session.cpp',
+    'Http3Stream.cpp',
     'HttpAuthUtils.cpp',
     'HttpBackgroundChannelChild.cpp',
     'HttpBackgroundChannelParent.cpp',
@@ -134,6 +136,7 @@
     '/netwerk/base',
     '/netwerk/cookie',
     '/netwerk/ipc',
+    '/netwerk/socket/neqo_glue',
     '/netwerk/url-classifier',
 ]
 